Review.md

```diff
@@ -3,10 +3,18 @@
 You can exercise your Google-Fu for this and any other _Sprint Challenge_ in the future.
 
 1.  Explain the difference between `RDBMS` and `SQL`.
+--RDBMS is a software such DB browser that lets you use a GUI to perform database actions as to where SQL lets you use a programming language to accomplish the same thing. The RDBMS may feel more natural to users as it can show more visual representation of the data at one time with less typing.
```

Comment:
If you wanted to format the answers as bullet points of sorts (and also get a line break with that), you'd do it like so:
```markdown
- RDBMS ....
```
which ends up being:
- RDBMS ...